pkg/sdk/session.go

```diff
@@ -0,0 +1,106 @@
+package sdk
+
+import (
+	"context"
+	"log"
+	"net/url"
+
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// sessionSuppliers is the structure that represents a session's end block height
+// and its matching suppliers.
+type sessionSuppliers struct {
```

Comment:
I noticed that `sessionSuppliers` is lowercase but it's params are exported. 

Can you please make sure that the structs and fields in this file all have the correct scope and uppercase/downcase appropriately?